resources/config/README.md

```diff
@@ -22,6 +22,8 @@ Note: at least one logging sink needs to be used
 ### module-paths:
 * key : number that corresponds to the module being loaded
 * value : path to the module shared library file
+### module-binary-path:
+  - path to the module binary for async function execution over shared memory. If none is provided, the module will be loaded as a shared library
```

Comment:
Update documentation and describe what is a ModuleBInary and thatthare are two ways o fusing modules in MG now.